eo-parser/src/main/java/org/eolang/parser/PhiSyntax.java

```diff
@@ -107,7 +107,7 @@ public XML parsed() throws IOException {
         final XML dom = Syntax.CANONICAL.pass(
             new XMLDocument(
                 new Xembler(
-                    new Directives(xel).append(spy).append(this.extra)
+                    new Directives(xel).append(spy.directives()).append(this.extra)
```

Comment:
@volodya-lombrozo looks like it should be `.append(new DrErrors(spy))`, isn't it?